packages/styles/badge.css

```
@@ -0,0 +1,44 @@
+:root {
+  --badge-text-color: var(--gray-90);
+  --badge-background-color: transparent;
+  --badge-border-color: var(--gray-30);
+
+  --badge-height: var(--button-height);
+  --badge-small-height: 1.5rem;
+
+  --badge-icon-size: 20px;
```

This should be inheriting from the default `--icon-size` property.

```suggestion
  --badge-icon-size: var(--icon-size);
```